opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -13739,7 +13740,7 @@ type Mutation {
   campaignAdd(input: CampaignAddInput!): Campaign @auth(for: [KNOWLEDGE_KNUPDATE])
   campaignEdit(id: ID!): CampaignEditMutations
   ############### Containers
-  containerEdit(id: ID!): ContainerEditMutations
+  containerEdit(id: ID!, workspaceId: ID): ContainerEditMutations
```

Comment:
I don't think you need to add `workspaceId` here

Reply:
Ok, let me try it

Reply:
You're right. I'll update the code